tests/test_variables.py

```diff
@@ -47,134 +48,153 @@ def test_rule_variable_decorator_internals(self):
+        self.assertTrue(string_var.is_rule_variable)
+        self.assertEqual(string_var.field_type, StringType)
+        self.assertEqual(string_var.label, "My Label")
 
     def test_string_rule_variable_no_parens(self):
         @string_rule_variable
-        def string_var(): pass
+        def string_var():
+            pass
 
-        self.assertTrue(getattr(string_var, 'is_rule_variable'))
-        self.assertEqual(getattr(string_var, 'field_type'), StringType)
+        self.assertTrue(string_var.is_rule_variable)
+        self.assertEqual(string_var.field_type, StringType)
 
     def test_boolean_rule_variable(self):
-        @boolean_rule_variable(label='My Label')
-        def boolean_var(): pass
+        @boolean_rule_variable(label="My Label")
+        def boolean_var():
+            pass
 
-        self.assertTrue(getattr(boolean_var, 'is_rule_variable'))
-        self.assertEqual(getattr(boolean_var, 'field_type'), BooleanType)
-        self.assertEqual(getattr(boolean_var, 'label'), 'My Label')
+        self.assertTrue(boolean_var.is_rule_variable)
+        self.assertEqual(boolean_var.field_type, BooleanType)
+        self.assertEqual(boolean_var.label, "My Label")
 
     def test_boolean_rule_variable_no_parens(self):
         @boolean_rule_variable
-        def boolean_var(): pass
+        def boolean_var():
+            pass
 
-        self.assertTrue(getattr(boolean_var, 'is_rule_variable'))
-        self.assertEqual(getattr(boolean_var, 'field_type'), BooleanType)
+        self.assertTrue(boolean_var.is_rule_variable)
+        self.assertEqual(boolean_var.field_type, BooleanType)
 
     def test_select_rule_variable(self):
-        options = {'foo': 'bar'}
+        options = {"foo": "bar"}
 
         @select_rule_variable(options=options)
-        def select_var(): pass
+        def select_var():
+            pass
 
-        self.assertTrue(getattr(select_var, 'is_rule_variable'))
-        self.assertEqual(getattr(select_var, 'field_type'), SelectType)
-        self.assertEqual(getattr(select_var, 'options'), options)
+        self.assertTrue(select_var.is_rule_variable)
+        self.assertEqual(select_var.field_type, SelectType)
+        self.assertEqual(select_var.options, options)
 
     def test_select_multiple_rule_variable(self):
-        options = {'foo': 'bar'}
+        options = {"foo": "bar"}
 
         @select_multiple_rule_variable(options=options)
-        def select_multiple_var(): pass
+        def select_multiple_var():
+            pass
 
-        self.assertTrue(getattr(select_multiple_var, 'is_rule_variable'))
-        self.assertEqual(getattr(select_multiple_var, 'field_type'), SelectMultipleType)
-        self.assertEqual(getattr(select_multiple_var, 'options'), options)
+        self.assertTrue(select_multiple_var.is_rule_variable)
+        self.assertEqual(select_multiple_var.field_type, SelectMultipleType)
+        self.assertEqual(select_multiple_var.options, options)
```

Comment:
Changed based on a warning from flake8 where it says getattr should only be used in special cases.